tests/integration_tests/api/test_http.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import requests
+import runpy
+
+from subprocess import Popen
+import time
+import os
+import signal
+from random import randint
+
+
+@pytest.fixture(scope="module", autouse=True)
+def ds_name():
+    rand = randint(0,pow(10,12))
+    return f'hr_ds_{rand}'
+
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
I think these don't need `autouse` as they are only in effect when you use them anyway